react/components/Table/SimpleTable.js

```diff
@@ -132,11 +132,8 @@ class SimpleTable extends Component {
           </div>
         ) : (
           <div>
-            <AutoSizer>
+            <AutoSizer key={updateTableKey}>
```

Comment:
maybe we should guarantee this is always a string. I dunno the behaviour react makes if a key is undefined or null
```suggestion
            <AutoSizer key={`vtex-table__${updateTableKey}`}>
```

Reply:
I've just tested it, it works like if you were not passing at all.

Reply:
LGTM then